### PR TITLE
Racing car: add gasless and signless

### PR DESCRIPTION
**Pull request by vraja-nayaka**

### CODE REVIEW

File: frontend/packages/signless-transactions/src/hooks/use-create-session.ts [hunk 1]
```diff
@@ -127,7 +131,9 @@ function useCreateSession(programId: HexString, metadata: ProgramMetadata | unde
     console.log('metadata.types?.others?.output');
     console.log(metadata.types?.others?.output);
 
-    const hexToSign = metadata.createType(metadata.types.others.output, payloadToSign).toHex();
+    const hexToSign = createSignatureType
```
Comment (vraja-nayaka) [inline frontend/packages/signless-transactions/src/hooks/use-create-session.ts:134]: this is a solution for flexibility of approach. 
For example in `car races` it was necessary to have a type tuple for `metadata.types.others.output`, therefore we need to get only the first element of the array

**Review by faizov — APPROVED** [review]

File: frontend/apps/racing-car-game/src/features/Game/component/Layout/Layout.tsx [hunk 2]
```diff
@@ -56,14 +61,14 @@ function LayoutComponent() {
 
   const getDecodedPayload = (payload: Bytes) => {
     if (meta?.types.others.output) {
-      return meta.createType(meta?.types.others.output, payload).toHuman();
+      return meta.createType(meta?.types.others.output, [null, payload]).toHuman();
```
Comment (nikitayutanov) [inline frontend/apps/racing-car-game/src/features/Game/component/Layout/Layout.tsx:64]:
```suggestion
      return meta.createType(meta?.types.others.output, [null, payload]).toHuman() as [null, DecodedReply];
```
I think better to make assertion locally in a function

File: frontend/apps/racing-car-game/src/features/Game/hooks.ts [hunk 3]
```diff
@@ -53,14 +50,14 @@ function useSubscribeSentMessage() {
 
   const getDecodedPayload = (payload: Bytes) => {
     if (meta?.types.others.output) {
-      return meta.createType(meta?.types.others.output, payload).toHuman();
+      return meta.createType(meta?.types.others.output, [null, payload]).toHuman();
```
Comment (nikitayutanov) [inline frontend/apps/racing-car-game/src/features/Game/hooks.ts:53]: Same as above. Actually, function is repeats itself, can move it to a shared folder.

**Review by nikitayutanov — APPROVED** [review]

File: frontend/packages/signless-transactions/src/hooks/use-create-session.ts (hunk 1 above)
Comment (nikitayutanov) [reply frontend/packages/signless-transactions/src/hooks/use-create-session.ts:134]: Good solution, let's leave a comment to describe the purpose of this prop.

**Review by nikitayutanov — APPROVED** [review]

**Pull request merged**